src/core/batchWriter/BatchWriter.ts

```diff
@@ -164,19 +164,19 @@ export class BatchWriter {
     await Promise.all(promises);
   }
 
-  private sendCreateBuffer (buffer: BatchBuffer, options?: { refresh?: 'wait_for' }) {
+  private sendCreateBuffer (buffer: BatchBuffer, options?: { refresh?: 'wait_for' | 'false( }) {
```

Comment:
```suggestion
  private sendCreateBuffer (buffer: BatchBuffer, options?: { refresh?: 'wait_for' | 'false' }) {
```
Seems like something went wrong while you did change the refresh types :sweat_smile: